src/app/modules/time-entries/pages/time-entries.component.spec.ts

```diff
@@ -247,7 +247,7 @@ describe('TimeEntriesComponent', () => {
         uri: 'abc',
       }, shouldRestartEntry: false
     };
-    component.entryId = 'new-entry';
+    component.entryId = 'entry_1';
```

Comment:
I'm not sure why we changed this value... entry_1 is another entry used in other tests.

Reply:
@josepato87 the if of `entry_1` is from  `component.activeTimeEntry = entry;` line 239

Reply:
Yep, but if you consider the purpose of the test is to throw error when something is wrong... we don't need to change the value... `new-entry` is ok for the context of the test, I believe.